connector/tests/src/test/java/org/oaebudt/edc/ManagementApiTransferTest.java

```diff
@@ -287,4 +344,55 @@ public JsonObject createCallback(final String url, final boolean transactional,
                 .build();
     }
 
+    private static String createParticipantManifest(int protocolPort, int credentialsPort, String participantId, String participantName)
+    {
+        String base64ParticipantId = encodeBase64(participantId);
+
+        return String.format("{\n" +
+                "    \"roles\": [],\n" +
+                "    \"serviceEndpoints\": [\n" +
+                "        {\n" +
+                "            \"type\": \"CredentialService\",\n" +
+                "            \"serviceEndpoint\": \"http://localhost:%d/api/credentials/v1/participants/%s\",\n" +
+                "            \"id\": \"%s-credentialservice-1\"\n" +
+                "        },\n" +
+                "        {\n" +
+                "            \"type\": \"ProtocolEndpoint\",\n" +
+                "            \"serviceEndpoint\": \"http://localhost:%d/protocol\",\n" +
+                "            \"id\": \"%s-dsp\"\n" +
+                "        }\n" +
+                "    ],\n" +
+                "    \"active\": true,\n" +
+                "    \"participantId\": \"%s\",\n" +
+                "    \"did\": \"%s\",\n" +
+                "    \"key\": {\n" +
+                "        \"keyId\": \"%s#key-1\",\n" +
+                "        \"privateKeyAlias\": \"%s#key-1\",\n" +
+                "        \"keyGeneratorParams\": {\n" +
+                "            \"algorithm\": \"EC\"\n" +
+                "        }\n" +
+                "    }\n" +
+                "}", credentialsPort, base64ParticipantId, participantName, protocolPort, participantName, participantId, participantId, participantId, participantId);
+    }
+
+    private static String encodeBase64(String input) {
+        return Base64.getEncoder().encodeToString(input.getBytes());
+    }
+
+    private static void createIdentityHubParticipant(OaebudtParticipant participant, String body, String apiKey) {
+        String apiUrl =  participant.getIdentityHubApiUri() + "/v1alpha/participants";
+
+        RequestSpecification request = RestAssured.given();
+        Response response = request
+                .header("x-api-key", apiKey)
+                .contentType(ContentType.JSON)
+                .log().all()
+                .body(body)
+                .post(apiUrl);
+
+        if (response.getStatusCode() != 200) {
+            System.err.println("Request failed with status: " + response.getStatusCode());
+            System.err.println("Response body: " + response.getBody().asString());
```

Comment:
please avoid System print at all. this behavior can be achieved by restassured directly, by concatenating, after the `post` call:
```
.log().ifValidationFails()
```